### DISCUSSION TITLE
OVH poor service!

### DISCUSSION THREAD

**Discussion by cassioemc** [ovh/manager#6081] [General]
I hired a VPS machine at OVH and they canceled access to the machine without the slightest notice. IT'S ABSURD WHAT THEY DID TO ME.
An error has occurred loading the page.

I opened two support tickets and so far I have not had any feedback. ALL MY DATA ARE IN THE SERVER. ABSURD.

ticket 3425475

**Comment by antleblanc** [comment]
Hi @cassioemc!

Unfortunately there's nothing we can do regarding your services from here.
Please contact our support team who will be able to help you or refer to https://help.ovhcloud.com.

Thank you!
